src/bundle/Resources/public/scss/_custom.scss

```diff
@@ -20,6 +22,7 @@ $ibexa-color-base-deep-dark: #131C26;
 
 $ibexa-color-font: #131C26;
 $ibexa-color-font-pale: #878B90;
+$ibexa-color-border-light: #ECDFD1;
```

Comment:
personally I would keep to one standard, either capitalize hex colors or not in whole project